### PR TITLE
Add Microchip

### LINKED ISSUE #8774: Add Microchip logo
### Brand Name

Microchip Technology Incorporated

### Website

https://www.microchip.com/

### Popularity Metric

The Similarweb rank is 33,556. See https://www.similarweb.com/website/microchip.com/

### Official Resources for Icon and Color

PNG: https://www.microchip.com/en-us/about/legal-information/microchip-trademarks
Color: `#E30620`
Logo and use guidelines: https://www.microchip.com/en-us/about/legal-information/website-terms-and-conditions

### Additional Comments

_No response_

### Contributing

Yes

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
microchiptechnologyincorporated

**Issue:** closes #8774 

**Similarweb rank:** 33,556. See https://www.similarweb.com/website/microchip.com/

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Hex Value picked from logo:  `#E30620`
Logo extracted from PNG found on brand website, auto-traced and cleaned up

### Tests
```
PS X:\~~~> npm test    

 22284 passing (985ms)

PS X:\~~~> npm run lint
--------------------- Summary ---------------------
✓ 2472 valid files.
```

### CODE REVIEW

**Comment by LitoMore** [comment]
@PratyushJayachandran The `®` sign needs to be removed.

**Comment by PratyushJayachandran** [comment]
> @PratyushJayachandran The `®` sign needs to be removed.

Fixed and pushed the commit

**Comment by PratyushJayachandran** [comment]
The logo had a minor misaligned control point. Fixed it.
microchiptechnologyincorporated

**Comment by LitoMore** [comment]
image

@PratyushJayachandran I used https://commons.wikimedia.org/wiki/File:Microchip_logo.svg as a reference. I can see some minor differences in it.

Would you mind using the SVG resource from Wikimedia to reprocess the icon?

I think it's better than vectorizing from the PNG.

**Comment by PratyushJayachandran** [comment]
> image
> 
> @PratyushJayachandran I used https://commons.wikimedia.org/wiki/File:Microchip_logo.svg as a reference. I can see some minor differences in it.
> 
> Would you mind using the SVG resource from Wikimedia to reprocess the icon?
> 
> I think it's better than vectorizing from the PNG.

I found the vectorized logo in this PDF from the microchip website http://ww1.microchip.com/downloads/en/DeviceDoc/00002523A.pdf
I will use this to push updates to this PR

**Comment by PratyushJayachandran** [comment]
@LitoMore Please review again. Source SVG extracted from http://ww1.microchip.com/downloads/en/DeviceDoc/00002523A.pdf
microchiptechnologyincorporated

File: _data/simple-icons.json [hunk 1]
```diff
@@ -7423,6 +7423,18 @@
             "hex": "FF8800",
             "source": "https://help.micro.blog"
         },
+        {
+            "title": "Microchip Technology Incorporated",
```
Comment (LitoMore) [inline _data/simple-icons.json:7427]:
```suggestion
            "title": "Microchip",
```

Comment (LitoMore) [reply _data/simple-icons.json:7427]: The `Microchip` is enough. There is no need to use the company's full name.

And those aliaes can be removed.

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: _data/simple-icons.json (hunk 1 above)
Comment (PratyushJayachandran) [reply _data/simple-icons.json]: I have fixed the JSON and SVG to match your suggestion.

**Comment by LitoMore** [comment]
> Any other use of a Trademark publicly may only be undertaken with advance written permission from Microchip. Nothing on this Site should be construed as granting any license or right to use any Trademark without the written permission of Microchip.

@PratyushJayachandran We must ask them for written permission.

**Pull request closed without merging**